recipe/meta.yaml

```diff
@@ -26,9 +26,9 @@ requirements:
     - python >={{ python_min }}
     - feedparser >=6.0.11
     - pytz
-    - requests >=2.21.0
     - bibtexparser
     - httpx >=0.28.1
+    - pytest-asyncio
```

Comment:
Is this a run requirement or only used for tests? If only for tests, maybe it does not need to be present here.

Reply:
Yep, it's test-only.

Reply:
You can move it to the test section below then.